src/main/utils/utils.ts

```diff
@@ -151,9 +151,9 @@ export class Utils {
         }
     }
 
-    public static removeDirIfExists(dirPath: string) {
+    public static async removeDirIfExists(dirPath: string) {
         if (fs.existsSync(dirPath)) {
```

Comment:
removeFolder already does the same as removeDirIfExists, we should remove code duplication here.

Reply:
done